controls/sidedrawer/overview.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Overview
+page_title: .NET MAUI SideDrawer Documentation | Overview
+description: Check our &quot;Overview&quot; documentation article for Telerik SideDrawer for .NET MAUI control.
+position: 0
+slug: sidedrawer-overview
+---
+
+# Overview
+
+`Telerik UI for .NET MAUI SideDrawer Control` is designed to enable users to visualize a hidden view in their applications. That view can host navigation UI, common settings or any other UI of your choice. It can be visualized using a flick gesture and can be shown from all four sides of the screen.
+
+![SideDrawer example](images/sidedrawer-overview.png) 
+
+## Key features
+
+* [Drawer positions]({%slug sidedrawer-features-configuration%})&mdash; The drawer can be shown from any of the four edges of the screen.  You can define its position through the DrawerLocation property.
+
+* [Effects and transitions]({%slug sidedrawer-features-transitions%})&mdash; SideDrawer supports transition animations that are applied to the opening and closing view. We provide several predefined animations that can be easily set or changed at runtime to meet our customers’ requirements.
+
+* [Drawer length]({%slug sidedrawer-features-configuration%})&mdash;
+
+* [Commands]({%slug sidedrawer-features-commands%})&mdash; SideDrawer allows you to attach commands that will be executed when certain actions, such as Opening and Closing occur.
+
+## See Also
+
+- [Getting Started with Telerik UI for .NET MAUI SideDrawer]({%slug sidedrawer-getting-started%}) 
```

Comment:
keep only the getting started slug, remove the others. Same as other controls overview articles